docs/source/VUDRM-token.md

```diff
@@ -0,0 +1,253 @@
+
+The `polbegin` and `polend` settings use the timezone set at the account level. Please contact support@vualto.com to confirm the timezone for your account. 
+
+There are also limitations depending on environments that are not explained in the table. Please refer to the following sections for more detail:
+
+### PlayReady DRM policy
+
+PlayReady has an extensive list of policy options described [here](https://docs.microsoft.com/en-us/playready/overview/license-and-policies). The majority of these options are supported via the VUDRM token's policy. Please contact support@vualto.com for more details.
+
+### Fairplay DRM policy
+
+Fairplay has three types of license; `rental`, `lease`, and `persist`. 
+
+Fairplay licenses can only be persisted past the user session by an offline enabled iOS application. When using Safari a session will be preserved until the browser tab is closed.
+
+Fairplay does not allow playback over non-HDCP connections. 
+
+#### Fairplay rental DRM policy
+
+You can specify a `rental` license by setting the `type` key to `r` in the DRM policy.
+
+You can set the expiry of the license using the `duration_rental` key or the `polend` key. If both `duration_rental` and `polend` are set in a policy then `duration_rental` will be used.
+
+Using the `type` of `rental` will not cause playback to stop at the license expiry if the encryption keys have not changed.
+
+#### Fairplay lease DRM policy
+
+You can specify a `lease` license by setting the `type` key to `l` in the DRM policy.
+
+You can set the expiry of the license using the `duration_lease` key or the `polend` key. If both `duration_lease` and `polend` are set in a policy then `duration_lease` will be used.
+
+Using the `type` of `lease` will cause playback to stop at the license expiry. Normally a player will make a new license request at this point, please ensure the VUDRM token is updated before further license requests are made.
+
+#### Fairplay persist DRM policy
+
+The use case of the `persist` license is for offline playback. 
+
+You can specify a `persist` license by setting the `type` key to `p` in the DRM policy. Setting `licache` to `yes` will also cause a persist license to be generated. If `type` and `liccache` are set in the DRM policy then the license type will be set to `persist`.
+
+You can set the expiry of the license using the `duration_persist` key or the `polend` key. If both `duration_persist` and `polend` are set in a policy then `duration_persist` will be used.
+
+Using the `type` of `persist` will cause playback to stop at the license expiry and the license will be removed from the device.
+
+### Widevine DRM policy
+
+Widevine licenses can only be persisted past the user session by an Android application. When using Chrome a session will be preserved until the browser tab is closed.
+
+### DRM policy examples
+
+The following sections display some example polices. These polices apply to all DRM providers.
```

Comment:
These don't look like examples. They look like the message format. I would expect to see real values in an example.
How would I define polend being 24hrs after polbegin?

Reply:
Will amend. Need to have some thought as to what examples we want to add.